README.md

```diff
@@ -1,2 +1,4 @@
-# community_commitments
-Corresponds with #community_commitments channel in Slack
+# Code with Asheville Community Commitments
+_Corresponds with #community_commitments channel in Slack_
```

Comment:
What do you think about linking to our Slack workspace from here?
It would be nice for discoverability, but I'm not sure whether we might subject ourselves to abuse from folks scanning public repos.